view/frontend/web/js/model/adyen-payment-service.js

```diff
@@ -65,6 +65,26 @@ define(
                         self.setPaymentMethods([]);
                     }
                 )
+            },
+            getOrderPaymentStatus: function (orderId, callback) {
+                var self = this;
+                var serviceUrl = urlBuilder.createUrl('/adyen/orders/:orderId/payment-status', {
+                    orderId: orderId
+                });
+
+                storage.get(
+                    serviceUrl
+                ).done(
+                    function (response) {
+                        if (callback) {
+                            callback(response);
+                        }
+                    }
+                ).fail(
+                    function (response) {
+                       console.log(response);
+                    }
```

Comment:
This means that the shopper will never know if an error occurs

Reply:
We should not do this indeed and log the error on server level.